### DISCUSSION TITLE
Cannot crop a crop

### DISCUSSION THREAD

**Discussion by howff** [ImageMagick/ImageMagick#4198] [Help]
### Prerequisites

- [ x] I have written a descriptive issue title
- [x ] I have verified that I am using the latest version of ImageMagick
- [x ] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
Cannot crop and image which is the result of a previous crop.

### Steps to Reproduce
magick.exe file.png -crop 100x100+100+100 crop1.png
magick.exe crop1.png -crop 100x100+100+100 crop2.png
sometimes you get an error magick: geometry does not contain image,
although I've also seen crops which are simply wrong,
or it writes out 200 sub-images(!!).

### System Configuration
Windows 10 ImageMagick-7.0.10-44-Q16-x64-static, but problem existed in previous versions too.

**Comment by snibgo** [comment]
After "-crop", IM records the canvas metadata. The second crop is with respect to that canvas, not the cropped image. To remove canvas metadata, use "+repage" after every crop.

In general, operations such as trim and distort create canvas metadata, so you should always consider following these with "+repage".
